### DISCUSSION TITLE
Bulk update records [Oct 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4097] [Discord Threads]
> 🙋‍♂️ **Mark Ambrazhevich**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### From UI, no.    
####    
#### Bulk update API's, you can refer here: https://docs.nocodb.com/developer-resources/rest-apis#data-apis  
#  
> 🙋‍♂️ **emdax**  
#### would be awesome as UI capability  
#  
> 🙋‍♂️ **Mark Ambrazhevich**  
#### That would be really desired feature  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1025217503871385670)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
